test/test_pebble.py

```diff
@@ -1481,21 +1481,21 @@ def test_get_services_names(self):
     def test_pull_text(self):
         self.client.responses.append((
             {'Content-Type': 'multipart/form-data; boundary=01234567890123456789012345678901'},
-            b"""
---01234567890123456789012345678901
-Content-Disposition: form-data; name="files"; filename="/etc/hosts"
-
-127.0.0.1 localhost  # """ + b'\xf0\x9f\x98\x80\nfoo\r\nbar' + b"""
---01234567890123456789012345678901
-Content-Disposition: form-data; name="response"
-
+            b"""\
+--01234567890123456789012345678901\r
+Content-Disposition: form-data; name="files"; filename="/etc/hosts"\r
+\r
```

Comment:
I don't understand how these `\r` and suchlike are working. Isn't `b"""` a raw string, meaning that there are no escape sequences, and `\r` will end up as a literal backslash followed by a literal `r`?

Reply:
Aha, @jameinel's right, sorry. `b` is byte, `r` is raw, and `"""` just means multiline (but still processes escapes). I had to type it into a Python REPL to re-convince myself. :-) I've been doing too much Go, where backtick strings are multiline *and* raw.